### PR TITLE
Add DaoCloud case study

### PR DESCRIPTION
**Pull request by windsonsea**
Add a case study of DaoCloud that promotes Istio spreading in China.

- [ ] Ambient
- [ ]  Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/about/case-studies/daocloud/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+skip_pagenav: true
+skip_feedback: true
+doc_type: article
+sidebar_force: sidebar_case_study
+type: case-studies
+weight: 52
+---
+
+Istio has gained recognition among DaoCloud's users for its excellent architecture and features,
+leading to improved business efficiency. As a leading cloud-native service provider in China,
```
Comment (Michelle951) [inline content/en/about/case-studies/daocloud/index.md:22]:
```suggestion
leading to improved business efficiency. As a leading cloud native service provider in China,
```
cncf uses "cloud native"， no hyphen.

File: content/en/about/case-studies/daocloud/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+DaoCloud offers high-quality services to customers across various industries. They utilize Istio
+as the foundation for their service mesh product, enabling customers to address specific needs
```
Comment (Michelle951) [inline content/en/about/case-studies/daocloud/index.md:24]:
```suggestion
as the foundation for their service mesh offerings, enabling customers to address specific needs
```
or use "products"

File: content/en/about/case-studies/daocloud/index.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+1. Enhanced observability: In multi-cloud environments, Istio's observability tools provide deep
+   insights into service behavior and performance, facilitating issue detection and troubleshooting.
+
+1. Unified security policies: Istio's security features, including mTLS and authorization policies,
+   ensure consistent security controls regardless of where the services are running.
+
+1. Unified traffic policies: Istio enables control over traffic between services,
+   adapting to diverse requirements such as active-active setups or disaster recovery scenarios.
+
+1. Inter-cluster network connectivity: Istio simplifies interconnecting different
```
Comment (Michelle951) [inline content/en/about/case-studies/daocloud/index.md:46]:
```suggestion
1. Inter-cluster network connectivity: Istio makes it easier to connect different
```

File: content/en/about/case-studies/daocloud/index.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+   more cloud-native architecture.
+
+1. Service discovery and traffic management: Istio provides robust service discovery
+   and traffic management capabilities, enabling effective management and control of services.
+
+1. Observability and monitoring: Istio's observability tools provide detailed insights
+   into service behavior and performance, facilitating issue identification and troubleshooting.
+
+1. Security and compliance: Istio's security features, such as mTLS authentication and
+   access control policies, ensure the enforcement of security and compliance requirements
```
Comment (Michelle951) [inline content/en/about/case-studies/daocloud/index.md:107]:
```suggestion
   access control policies, ensuring the enforcement of security and compliance requirements
```

**Comment by Michelle951** [comment]
lgtm

File: content/en/about/case-studies/daocloud/index.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+Canary deployment is a software release model where new versions are initially released to
+a small subset of users instead of all users. This way, any issues with the new version only
+impact a limited number of users. Once the new version is confirmed to be problem-free,
+it can gradually roll out to all users.
+
+Istio's service mesh simplifies the implementation of canary deployment models.
+By utilizing Istio's virtual services and destination rules, specific traffic can
+be directed to the new version of the service. For example, only 10% of the traffic
+can be routed to the new version while the remaining 90% continues to use the old version.
+This enables thorough testing of the new version's features and performance within a small user group.
```
Comment (hanxiaop) [inline content/en/about/case-studies/daocloud/index.md:71]:
```suggestion
This enables thorough testing of the new version's features and performance, ensuring a low impact on the existing system.
```

File: content/en/about/case-studies/daocloud/index.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+   and traffic management capabilities, enabling effective management and control of services.
+
+1. Observability and monitoring: Istio's observability tools provide detailed insights
+   into service behavior and performance, facilitating issue identification and troubleshooting.
+
+1. Security and compliance: Istio's security features, such as mTLS authentication and
+   access control policies, ensuring the enforcement of security and compliance requirements
+   during the migration process.
+
+By leveraging Istio's capabilities, DaoCloud has successfully assisted numerous Chinese enterprises
```
Comment (hanxiaop) [inline content/en/about/case-studies/daocloud/index.md:110]:
```suggestion
Leveraging Istio's capabilities, DaoCloud has successfully assisted numerous Chinese enterprises
```

**Comment by craigbox** [comment]
Hi Michael, thanks for the submission.  It can be previewed [here](https://deploy-preview-13543--preliminary-istio.netlify.app/latest/about/case-studies/daocloud/) for reference.

Generally, we use this section for end user case studies.  This is more a service provider case study, which isn't necessarily a blocker but isn't what we've published so far.

Could you rewrite this from the angle of one of the Chinese enterprises you've helped, but you can leave in the fact that DaoCloud helped out?

**Comment by windsonsea** [comment]
Ok, I'll have a try to collect more detailed materials and rewrite it.

**Pull request closed without merging**